Makefile

```diff
@@ -470,6 +470,11 @@ install_data : $(DATA_INSTALL_DIR) $(DATA_SRC_DIR)
 # directories must be set to be owned by the calling user.
 ifdef SUDO_USER
 ifeq ("$(USER)","root")
+
+	# generate_selfcertificate creates the etc directory owned by root if it doesn't exists
+	# the data and etc directories must be set to be owned by the calling user.
```

Comment:
We already have this comment for main ifdef block

Reply:
done